clang/test/SemaSYCL/call-to-undefined-function.cpp

```diff
@@ -95,21 +97,22 @@ int main() {
   undefined();
 
   deviceQueue.submit([&](sycl::handler &h) {
-    h.single_task<class CallToUndefinedFnTester>([]() {
-      // expected-note@-1 {{called by 'operator()'}}
-      // expected-note@-2 {{called by 'operator()'}}
+    h.single_task<class CallToUndefinedFnTester>([]() { // #CALLOP
 
       // simple functions
       defined();
       undefinedExternal();
       undefined();
       // expected-error@-1 {{SYCL kernel cannot call an undefined function without SYCL_EXTERNAL attribute}}
+      // expected-note@#UNDEFINED {{'undefined' declared here}}
```

Comment:
What the heck is this feature? I've never seen this before. :-D

Reply:
Its a fun one!  https://clang.llvm.org/doxygen/classclang_1_1VerifyDiagnosticConsumer.html has a bunch of really cool features.

See this:
````
As an alternative to specifying a fixed line number, the location of a diagnostic can instead be indicated by a marker of the form "#<marker>". Markers are specified by including them in a comment, and then referenced by appending the marker to the diagnostic with "@#<marker>":
#warning some text  // #1
// expected-warning@#1 {{some text}}
````

I suggested (and I think it has been done a bit!) that the sycl.hpp testing shim use these!

Reply:
THIS IS SO COOL! I mean, uh.. neat, thanks. ;-)